mesmerize_napari/tests/test_core.py

```diff
@@ -73,8 +73,8 @@ def download_data(fname: str):
     return Path(path)
 
 
-def teardown_module():
-    clear_tmp()
+# def teardown_module():
```

Comment:
I'm assuming you're doing this to save time with tests while doing it locally? I think you can create an option to pass to pytest to not clear temp:
https://docs.pytest.org/en/6.2.x/example/parametrize.html

Create a file called `conftest.py`
```python
def pytest_addoption(parser):
    parser.addoption("--keep-test-data", action="store_true", help="keep test data after running tests")

def pytest_generate_tests(metafunc):
    if "keep_test_data" in metafunc.fixturenames:
        if metafunc.config.getoption("keep-test-data"):
            k = True
        else:
            k = False
        metafunc.parametrize("keep_test_data", k)
```

And in `test_core.py` make the `clear_tmp()` function like:

```python
def clear_tmp(keep_test_data):
    if keep_test_data:
        return
    else:
        # cleanup stuff
```

And then you can run pytest like this:

```
pytest --keep-test-data
```

Reply:
running pytest from terminal on laptop isn't working, I'm using pycharm ide. I'll do this when I test on nellab desktop